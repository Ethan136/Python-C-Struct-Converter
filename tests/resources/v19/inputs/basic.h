struct Customer {
    int Id;
    int Name;
};

