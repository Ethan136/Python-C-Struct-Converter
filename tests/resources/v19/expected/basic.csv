entity_name,field_order,field_name,data_type,offset,size,value,hex_raw
Customer,1,Id,int,0,4,1234,000004d2
Customer,2,Name,int,4,4,1,00000001
